examples/bootloader/.gitignore

```diff
@@ -0,0 +1 @@
+/keys.c
```

Comment:
Ok the issue was to automatically add the C file as a source to be compiled with the main.c source.